src/enumValues.ts

```diff
@@ -7,6 +7,11 @@ export class EnumValues {
     return Object.keys(e).filter(key => isNaN(+key))
   }
 
+  static getNameFromValue(e:any, value:(number|string)) {
+    const all = this.getNamesAndValues(e).filter(pair => { return pair.value == value;} );
```

Comment:
change the `{ return pair.value == value;}` to  `pair.value === value`
use the `===` instead of `==` and remove the `{}` and the `return`